Cult stun nerf + Zealots gives SecHuds (This time without 70 Commits)

JohnFulpWillard opened the pull request:
## About The Pull Request

This nerfs the cult's stun ability, causing mindshielded people to drop held items, jitter/stutter, bleed 5% of their blood and take 60 stamina damage (which means 2 consecutive cult stuns will result in stamcrit), while keeping the actual stun for non mindshielded people.
To slightly balance this out we've made a Fulp-only version of the Zealot's blindfold which has Security huds so they can tell who is mindshielded.

## Why It's Good For The Game

The cult stun should be for early game converting and not for winning fights against armed Security Officers lategame.
Also, unless the cult's target is the Chaplain, all it takes is one click to take your target out. This way, even if their target isn't the Chaplain, it still won't just take one click to win.
/tg/ justified that this nerf was bad because "Cult doesn't have security huds", so the blindfold is my compromise
They can also still put you into stamcrit if they use it twice, but that means they're missing another spell they could've had, which at least makes it slightly more balanced.

## Changelog
:cl:
balance: Cult stun causes mindshielded people to jitter rather than stun
balance: Zealot's blindfold gives SecHUDs
/:cl:

JohnFulpWillard committed:
Brings back cult stun nerf

The cult stun nerf was good, it should only be used in the early game for converting and shouldn't be an option to 1v3 officers and the cult's target is extremely easy if it isn't the Chaplain. This at least requires them to put in a little more effort, and gives the crew a reason to not remove their mindshield and resist getting any

JohnFulpWillard committed:
Revert "Brings back cult stun nerf"

This reverts commit 17b185eb54893183d6acc8bcfb4db65332a901b1.

JohnFulpWillard committed:
Merge branch 'master' of https://github.com/TomFulpWillard/Fulpstation

JohnFulpWillard committed:
Summary

description

ATH1909 commented on code/zFulpstationCode/fulp_cult_overwrite.dm:
```diff
@@ -0,0 +1,83 @@
+	name = "zealot's blindfold"
+	icon_state = "blindfold"
+	inhand_icon_state = "blindfold"
+	hud_type = DATA_HUD_SECURITY_ADVANCED
```
can you give this both a medHUD and a secHUD? the zealot's blindfold's abiltiy to display health bars is really nice to have if you're using things other than the gamer hand.

JohnFulpWillard replied:
im a stupid god damn idiot but it's done now

JohnFulpWillard committed:
I'm stupid but good enough

Gave Zealots both huds and fixes non cultists being able to use them.  There was likely an extremely easy other way to do it, but I'm too stupid.

JohnFulpWillard committed:
Fixing stupid idiot code a little

ATH1909 commented:
Thanks for taking my suggestion, John!

Fikou commented on code/zFulpstationCode/fulp_cult_overwrite.dm:
```diff
@@ -0,0 +1,119 @@
+
+//This is a Fulp-only version of Zealot's blindfold, which comes with SecurityHUDs so Cultists can tell who is Mindshielded.
+
+/obj/item/clothing/glasses/hud/health/night/fulpcultblind
```
why are these still medhuds if you made special code for them to give sec huds AND med huds

JohnFulpWillard replied:
because they didnt actually show up in game for some reason, unless that's just for me
